C/plugins/storage/postgres/connection.cpp

```diff
@@ -3312,3 +3314,712 @@ const char *p;
+ 
+										}
+										else
+										{
+                                                                			c.column = v["column"].GetString(); 
+										}
+									}
+
+									if (v.HasMember("type"))
+									{
+										if (!v["type"].IsString())
+										{
+											Logger::getLogger()->error("%s %d ", __FUNCTION__, __LINE__, "extracting type, expecting a string value here");
+										}
+										else
+										{
+											c.type = v["type"].GetString();
+											if (c.type == "double") c.type = "real";
+										}
+									}
+
+									if (v.HasMember("size"))
+									{
+										if(!v["size"].IsInt())
+										{
+											Logger::getLogger()->error("%s %d extracting size, expecting an int value here", __FUNCTION__, __LINE__);
+										}
+										else
+										{
+											c.sz = v["size"].GetInt();
+										}
+									}
+
+									if (v.HasMember("key"))
+									{
+										if(!v["key"].IsBool())
+                                                                                {
+											Logger::getLogger()->error("%s %d extracting key, expecting a bool value here", __FUNCTION__, __LINE__);
+
+                                                                                }
+                                                                                else
+                                                                                {
+                                                                                         c.key = v["key"].GetBool();
+                                                                                }
+                                                                        }
+
+									colsPerTableInReq.insert(c);
+								}
+							}
+
+							// Iterate over all the indexes per table and store in indexesMatrixFromReq
+							Value& idx = tables[i]["indexes"];
+							if (!idx.IsArray())
+                                                        {
+                                                                Logger::getLogger()->error("The property indexes must be an array");
+                                                        }
+							else
+							{
+                        					for (auto& v : idx.GetArray())
+                        					{
+                                        				std::vector<std::string> indexVec;
+									std::string s;	
+				                                        if (v.IsObject())
+                                        				{
+				                                                if (v.HasMember("index"))
+                               					                {
+                                                        				if (!v["index"].IsArray())
+                                                        				{
+				                                                                Logger::getLogger()->error("%s %d ", __FUNCTION__, __LINE__, "extracting index values , expecting an array here");
```

Comment:
Help the user know which schema, service, table....

Reply:
Fixed